### PR TITLE
docs: Update README with more information about the framework

### PR DESCRIPTION
**Pull request by trevanhetzel**
Adding an introduction and more details about the features available.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
+- **Smart content redirects**
+  - Automatically redirects content from the WP site to the front-end site to minimize site visitors’ confusion and avoid SEO penalties for duplicate content
+  - Redirects hyperlinks inserted into posts’ content to the front-end site
+- **Removal of WP themes**
+  - Prevents confusion around the active theme
+- **Ability to define custom menus in a GUI**
+- **Additional data exposed through WPGraphQL**
+  - Block stylesheets
+
+### npm Package Features
```
Comment (trevanhetzel) [inline README.md:38]: Need to flesh the npm features out more for sure

File: README.md [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
-Eager to try out the Headless Framework? Here's how you can get started with our Preview example:
+Eager to try out the Headless Framework? Here's how you can get started:
 
 1. Create a WordPress site if you haven't already. We recommend using [Local](https://localwp.com/)!
-2. Download, upload, and activate the `wpe-headless` plugin in this repository. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
+2. Download, upload, and activate the `wpe-headless` plugin. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
 3. Install [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) on the WordPress site if it's not already installed
-4. [Clone this repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) to a directory of your choice
-5. Navigate to `examples/preview` in the cloned repository
+4. Download our [Getting Started example project](https://github.com/wpengine/headless-framework/tree/canary/examples/getting-started) and navigate to it locally
```
Comment (nickcernis) [inline README.md:17]: `getting-started` doesn't exist yet.

Should we leave this URL as https://github.com/wpengine/headless-framework/tree/canary/examples/preview until `getting-started` exists? @claygriffiths mentioned on Friday that we planned to rename `preview`.

File: README.md [hunk 3]
```diff
@@ -1,32 +1,50 @@
...
 6. `cp .env.local.sample .env.local`
 7. Populate `NEXT_PUBLIC_WORDPRESS_URL` and `WPE_HEADLESS_SECRET` accordingly in `.env.local`
 8. `npm install && npm run dev`
```
Comment (nickcernis) [inline README.md:20]: Perhaps make it more clear what `NEXT_PUBLIC_WORDPRESS_URL` should contain, and how to find the `WPE_HEADLESS_SECRET`:

```suggestion
6. `cp .env.local.sample .env.local`
7. Populate `NEXT_PUBLIC_WORDPRESS_URL` in `.env.local` with the full URL to your WordPress site, including the `http://` or `https://` prefix.
8. Populate the `WPE_HEADLESS_SECRET` in `.env.local` with the secret key found at Settings → Headless in your WordPress admin area.
9. `npm install && npm run dev`
```

File: README.md [hunk 4]
```diff
@@ -1,32 +1,50 @@
-# WordPress Headless Framework (PREVIEW/ALPHA)
+# Headless WordPress Framework
+
+## Introduction
+
+WPEngine's Headless WordPress Framework provides a set of tools to make building front-end applications with WordPress as the headless CMS a pleasant experience for both developers and publishers. This framework consists of a WordPress plugin, a set of npm packages, and guides to get you started building headless WordPress sites in Next.js.
```
Comment (nickcernis) [inline README.md:5]: 'WPEngine' → 'WP Engine' (to follow brand guidelines).

```suggestion
WP Engine's Headless WordPress Framework provides a set of tools to make building front-end applications with WordPress as the headless CMS a pleasant experience for both developers and publishers. This framework consists of a WordPress plugin, a set of npm packages, and guides to get you started building headless WordPress sites in Next.js.
```

File: README.md [hunk 5]
```diff
@@ -1,32 +1,50 @@
...
+- **Removal of WP themes**
+  - Prevents confusion around the active theme
+- **Ability to define custom menus in a GUI**
+- **Additional data exposed through WPGraphQL**
+  - Block stylesheets
+
+### npm Package Features
+- Auth handler for Express/Next that exchanges a code for an access token
+  - The access token can be used to make authenticated calls to WordPress via WPGraphQL or REST
+- `HeadlessProvider`
```
Comment (nickcernis) [inline README.md:41]:
```suggestion
- A `HeadlessProvider` component to ease communication with WordPress via [Apollo](https://www.apollographql.com/) and [WPGraphQL](https://www.wpgraphql.com/).
```

File: README.md [hunk 6]
```diff
@@ -1,32 +1,50 @@
...
+- `TemplateProvider`
```
Comment (nickcernis) [inline README.md:42]:
```suggestion
- A `TemplateLoader` component to load page templates based on the current URL path and page type.
```

File: README.md (hunk 1 above)
Comment (nickcernis) [reply README.md:38]: I added some suggested descriptions for `HeadlessProvider` and `TemplateLoader` below.

Comment (trevanhetzel) [reply README.md:38]: Thank you!

File: README.md (hunk 2 above)
Comment (trevanhetzel) [reply README.md:17]: Ah yeah, I forgot we talked about that. That's a much better experience than manually donloading.

Comment (trevanhetzel) [reply README.md:17]: Also, yeah I think we should rename `preview` to `getting-started`. For this PR though, yeah you're right I'll revert back to `preview` and we can update to `getting-started` once we make that change

File: README.md (hunk 3 above)
Comment (trevanhetzel) [reply README.md:20]: Good callouts, thanks!

**Review by matt-landers — APPROVED** [review]
Looks good!

File: README.md [hunk 7]
```diff
@@ -1,32 +1,55 @@
...
-Eager to try out the Headless Framework? Here's how you can get started with our Preview example:
+Eager to try out the Headless Framework? Here's how you can get started:
 
 1. Create a WordPress site if you haven't already. We recommend using [Local](https://localwp.com/)!
-2. Download, upload, and activate the `wpe-headless` plugin in this repository. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
-3. Install [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) on the WordPress site if it's not already installed
-4. [Clone this repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) to a directory of your choice
-5. Navigate to `examples/preview` in the cloned repository
+2. Download, upload, and activate the `wpe-headless` plugin. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
+3. Install [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) on the WordPress site if it's not already installed\
```
Comment (mindctrl) [inline README.md:16]:
```suggestion
3. Install [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) on the WordPress site if it's not already installed
```

Comment (trevanhetzel) [reply README.md]: Ope, good catch

**Pull request merged**